portfolio/src/main/webapp/index.html

```diff
@@ -24,22 +23,22 @@ <h1 id="intro">Hi. I'm Zian.</h1>
                         <p class="path-stop-desc">Thanks for dropping by! My name is Zian Chen, a soon-to-be third year student at UCLA, working to get a Bachelor's in Computer Science. My path to where I am now has not been easy, but along the way I've picked up multiple different skills and, arguably more importantly, how to deal with failure, learn the value of criticism, and rely not only on myself but those around me. Keep scrolling to track a few stops along the way! </p>
```

Comment:
Usually when the line it getting too long it's hard for people to read. Prefer to break lines in the case.
https://developers.google.com/style/html-formatting